### DISCUSSION TITLE
Does PGSYNC work with read replica of GCP hosted PostgreSQL database?

### DISCUSSION THREAD

**Discussion by AnkitKundariya** [toluaina/pgsync#387] [Q&A]
The goal here is to let pgsync connect with the read replica of the master database and use the read replica for pgsync instead of the writable master database.

Some points where pgsync might not work.

- It created triggers for tables which couldn't possible with read replica.
- It uses wal_log and replication slots which might not be supported by GCP hosted database.

@toluaina 

If I'm mistaken, please point out how a read replica of a master database might be used with pgsync.

**Comment by toluaina** [comment]
I don't think this will work. 
PGSync needs writs access to create triggers and setup replication slots to the target db.
A read replica prohibits this.
